sdk/storage/Azure.Storage.Files.Shares/src/Models/ShareFileUploadRangeOptions.cs

```diff
@@ -30,5 +30,11 @@ public class ShareFileUploadRangeOptions
         /// hashing on uploads.
         /// </summary>
         public UploadTransactionalHashingOptions TransactionalHashingOptions { get; set; }
+
+        /// <summary>
+        /// Optional.  Specifies if the file last write time should be set to the current time,
+        /// or the last write time currently associated with the file should be preserved.
+        /// </summary>
+        public FileLastWrittenMode? FileLastWrittenOnMode { get; set; }
```

Comment:
this property is named `FileLastWrittenOnMode` but the version of these options for "from uri" method calls it `FileLastWrittenMode`. They both use the same enum type. Is this intentional? Feels like it should be unified.

Reply:
Good catch, fixed.